h2o-algos/src/main/java/hex/pca/PCA.java

```diff
@@ -59,9 +57,12 @@
     double p = hex.util.LinearAlgebraUtils.numColsExp(_train,true);
     double r = _train.numRows();
     long mem_usage =
-            _parms._pca_method == PCAParameters.Method.GramSVD ? (long)(hb._cpus_allowed * p*p * 8/*doubles*/ *
+            (_parms._pca_method == PCAParameters.Method.GramSVD) || (_parms._pca_method==PCAParameters.Method.Power)
```

Comment:
Introduce variables for _parms._pca_method == PCAParameters.Method.GramSVD and _parms._pca_method==PCAParameters.Method.Power - why repeat the same phrase again and again?